### PR TITLE
fix: Import multiple segmentations from coco format json

### PR DESCRIPTION
**Pull request by Ljung323**
# Issue
https://github.com/HumanSignal/label-studio-sdk/issues/246

# Changes
- d5287ad pass all segmentation items to `create_segmentation` function
- 3b76299 fix the content of help

# Test

### CODE REVIEW

File: src/label_studio_sdk/converter/imports/coco.py [hunk 1]
```diff
@@ -270,7 +271,7 @@ def add_parser(subparsers):
         "--input",
         dest="input",
         required=True,
-        help="directory with COCO where images, labels, notes.json are located",
+        help="input COCO json file",
```
Comment (Ljung323) [inline src/label_studio_sdk/converter/imports/coco.py:274]: fixed to be consistent with: (not discussed in the original issue though)
https://github.com/HumanSignal/label-studio-sdk/blob/a67c59c724aaed8e0abdd368e4dea07306c78398/src/label_studio_sdk/converter/imports/coco.py#L120

File: src/label_studio_sdk/converter/imports/coco.py [hunk 2]
```diff
@@ -48,10 +48,9 @@ def create_bbox(annotation, categories, from_name, image_height, image_width, to
 
 
 def create_segmentation(
-    annotation, categories, from_name, image_height, image_width, to_name
+    category_id, segmentation, categories, from_name, image_height, image_width, to_name
```
Comment (Ljung323) [inline src/label_studio_sdk/converter/imports/coco.py:51]: due to the change from this PR, there's no need to pass the whole annotation.
so updated to pass `annotation["category_id"]` and `segmentation`

File: src/label_studio_sdk/converter/imports/coco.py [hunk 3]
```diff
@@ -216,15 +215,17 @@ def convert_coco_to_ls(
...
-            item = create_segmentation(
-                annotation,
-                categories,
-                segmentation_from_name,
-                image_height,
-                image_width,
-                to_name,
-            )
-            task[out_type][0]["result"].append(item)
+            for single_segmentation in annotation["segmentation"]:
```
Comment (Ljung323) [inline src/label_studio_sdk/converter/imports/coco.py:218]: to avoid conflicts with other `segmentation` variables, named it as `single_segmentation`

**Comment by makseq** [comment]
Could you please update your branch and merge the lastest master branch from the main repo?

**Comment by makseq** [comment]
Is it possible to add test for coco import? Something like this: https://github.com/Ljung323/label-studio-sdk/blob/master/tests/custom/converter/test_import_yolo.py

You can merely run the converter on a real simple COCO json, get its output and use it for test assertion. It would be wonderful to have it.

**Review by makseq — CHANGES_REQUESTED** [review]
1. Please update to the latest master branch
2. Could you add a test?

**Comment by Ljung323** [comment]
@makseq 
Thank you very much for the review.

I updated the code:

> 1. Please update to the latest master branch

92f6927

> 2. Could you add a test?

a3c8ba7 da9abcb

**Review by makseq — APPROVED** [review]

**Pull request merged**